### ISSUE TITLE
More member role options or allow granular, custom permissions for team members

### ISSUE THREAD

**Issue by vietqhoang** [heroku/roadmap#326]
### Required Terms

- [X] I agree to follow this project's [Code of Conduct](https://github.com/heroku/roadmap/blob/main/CODE_OF_CONDUCT.md)
- [X] I have read and accept the [Salesforce Program Agreement](https://www.salesforce.com/company/program-agreement/)

### What service(s) is this request for?

Heroku Pipelines, Heroku Apps, etc

### Tell us about what you're trying to solve. What challenges are you facing?

I have read the help documentation on user roles: https://help.heroku.com/CGNGRPE8/how-do-i-configure-who-receives-invoice-emails-from-heroku

I would love to see a team member role that has "billing" permissions; where the member can only access billing related information (payment method, invoices, etc) but does not have access to everything else.

For example the only way to access billing related details (payment methods, invoices, etc) is to be assigned the Admin role. The problem with this is that the Admin role has too much permission power in that they are able to manipulate the applications and other member roles.

Some organizations, like ours, the billing is handled by teams that are not engineer related. The existing available roles are very engineer-specific. It is a security risk to give someone in HR or billing the role of what Heroku defines as Admin.

TL;DR: Desire for more granular roles, specifically a role with only permissions to access billing.

**Comment by andre5oto** [comment]
@vietqhoang thank you for your suggestion. We are going to begin planning on more granular permissions for the upcoming year (2025). Stay tuned.
